frontends/web/src/components/update/update.tsx

```diff
@@ -30,13 +31,14 @@ interface File {
     description: string;
 }
 
-interface LoadedProps {
+interface Props {
     file: File | null;
 }
 
-type Props = LoadedProps & TranslateProps;
-
-function Update({ file, t }: PropsWithChildren<Props>): JSX.Element | null {
+const Update: FunctionComponent<Props> = ({ file }) => {
```

Comment:
what's the advantage of an arrow function over `function Update(...)`?

Reply:
here is a summary of differences: https://dmitripavlutin.com/differences-between-arrow-and-regular-functions/, but in a more simple answer, regular functions carry historical features that were introduced for `object prototyping`, arrow functions are more limited functional approach.